data/mods/Aftershock/mobs/mutants.json

```diff
@@ -54,5 +54,13 @@
         "monster_message": "A burst of frozen air explodes from their lungs!"
       }
     ]
+  },
+  {
+    "id": "mon_flesh_golem_afs",
+    "type": "MONSTER",
+    "copy-from": "mon_flesh_golem",
+    "upgrades": { "half_life": 15, "into": "mon_flesh_golem_afs" },
```

Comment:
Maybe replace it with something like
`"delete": { "upgrades": { "half_life": 38, "into": "jabberwock" } },`
Make sure the values fit the original mob

Reply:
I imagine the half life is there to upgrade existing flesh golems?

Reply:
The half-life is, I would think, there to overwrite the one for existing flesh golems, so this one doesn't transform into a jabberwock and spoil the mission.

Reply:
@Mrzjadacz: Does "delete" work (consistently) for things other than flags?

Reply:
Currently, he overrides 'upgrade' of the old mob with "upgrade into itself". It does nothing to non-afs flesh golems.

I run a quick search and there are several cases of using "delete": "upgrade..." in the game, but I didn't check in the code whether it works.